frontend/src/app/main/ui/workspace/left_header.cljs

```diff
@@ -108,6 +112,25 @@
          {:class (stl/css :file-name)
           :title file-name
           :on-double-click start-editing-name}
+          ;;-- Persistende state widget
+         [:div {:class (case (mf/deref ref:persistence-status)
```

Comment:
mf/deref is a react hook and should be statically defined in the let.
If this code on a future, for some unrelated reason is put under a conditional, it will break the app; hooks should be always defined statically.